ramls/data-import-converter-storage.raml

```diff
@@ -502,10 +502,16 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
-  /profileSnapshots/{jobProfileId}:
+  /profileSnapshots/{profileId}:
     displayName: Profile Snapshot structure
     get:
-      description: Get a profile snapshot structure for specified profile id
+      description: Get a profile snapshot structure for specified profile id and type
+      queryParameters:
+        profileType:
+          description: Indicates profile type
+          required: true
+          type: string
+          example: "JOB_PROFILE, ACTION_PROFILE, MATCH_PROFILE, MAPPING_PROFILE"
```

Comment:
is there a way to specify enum type for query param? if not - please move all the possible values to the description part and leave only one value as an example